microsoft-edge/devtools-guide-chromium/accessibility/reference.md

```diff
@@ -78,54 +78,10 @@ This article lists typical accessibility aspects to test for webpages, and the c
 | Verify that the webpage layout is usable when narrow | **Device Emulation** tool | [Verify that the webpage layout is usable when narrow](./narrow.md), and [Emulate mobile devices (Device Emulation)](../device-mode/index.md) |
 
 
-<!-- ====================================================================== -->
-## Further automated and manual testing
-
-Testing the accessibility of your website is important to ensure that people with different needs can use your website.  The DevTools features described above are a good start to catch accessibility problems in your products.  These features range from automated checks and manual detail checks to simulation of different states and environments.
-
-
-#### First, use DevTools automated reports and manual testing
-
-As covered in the tables above, DevTools includes automated accessibility-testing features, such as tools that automatically generate accessibility reports for a webpage, including the **Issues** tool and the **Lighthouse** tool.  You can also use the Assessments feature of Accessibility Insights, per below.
-
-Microsoft Edge, including DevTools, also supports manual accessibility testing, such as: 
-* Inspect different parts of the page by using the **Inspect** tool.
-* Use the keyboard to navigate the page.
-* Look for issues that arise when interacting with the page.
-* Look for issues related to changes in display, such as making the window narrow.
-
-
-#### Perform additional checks
-
-After the tests that are listed above, you may need to perform additional checks, such as:
-
-* Testing when zoomed-in.
-* Testing with screen readers.
-* Testing with voice recognition.
-* Testing in high-contrast mode.
-
-
-#### Use testers who have different accessibility needs
-
-Ideally, have testers with different accessibility needs use these automated testing features and do manual accessibility testing.
-
-Automated tools can't find all the problems in a product, because many of the accessibility barriers show up only during interactive use.  None of these features can replace a proper round of testing with people that use assistive technologies and following a plan to check for all the required tests.
-
-
-#### Use Accessibility Insights
-
-You can also use the [Assessments](https://accessibilityinsights.io/docs/en/web/getstarted/assessment/) feature of [Accessibility Insights](https://accessibilityinsights.io).
-
-
-#### Use webhint in Visual Studio Code
```

Comment:
@captainbrosset - did you deliberately remove section "Use webhint in Visual Studio Code"?

Reply:
Yes, webhint is available through our DevTools VSCode extension.